.github/wokrflows/npm-publish.yml

```
@@ -0,0 +1,34 @@
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          registry-url: https://registry.npmjs.org/
+      - run: npm i -f
+        env:
+          NPM_TOKEN: ${{ secrets.npm_token }}
+      - run: npm publish --access public
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.npm_token}}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)